build_env.rst

```diff
@@ -245,73 +245,78 @@ Remember to use ``-E`` option to pass the value of
+#. Any default values for CLI options that have not been overridden on the
+   command line, or by corresponding environment variables, will then take
+   effect.
 
 Defaults
 ========
 
-The following variables have defaults that can be customized by you via
-environment variables at runtime.
+The following variables have defaults that can be overridden by assigning your
+own values to the corresponding environment variables at runtime:
 
 Docker
 ------
 
-**{ENVPREFIX}_DOCKER_LOGIN** - Used for the interactive login for Docker
-Hub.
+| ``{ENVPREFIX}_DOCKER_LOGIN``
+| Set this to login to a Docker Repository interactively.
 
-**{ENVPREFIX}_DOCKER_USERNAME** - Your Docker username.
+| ``{ENVPREFIX}_DOCKER_USERNAME``
+| Your Docker username.
 
-**{ENVPREFIX}_DOCKER_PASSWORD** - Your Docker password.
+| ``{ENVPREFIX}_DOCKER_PASSWORD``
+| Your Docker password.
 
-**RUNSCRIPT_COMMAND** - Is not obtained from the environment, but is a
-hard coded default ("/bin/bash". This is the fallback command used in
-the case that the docker image does not have a CMD or ENTRYPOINT.
-**TAG** Is the default tag, ``latest``.
+| ``{ENVPREFIX}_COMMAND``
+| Is not obtained from the environment, but is a hard coded default
+  ("/bin/bash"). This is the fallback command used in the case that the docker
+  image does not have a CMD or ENTRYPOINT. ``TAG`` Is the default tag,
+  ``latest``.
 
-**{ENVPREFIX}_NOHTTPS** - This is relevant if you want to use a registry
-that doesn't have https, and it speaks for itself. If you export the
-variable ``{ENVPREFIX}_NOHTTPS`` you can force the software to not use
-https when interacting with a Docker registry. This use case is
-typically for use of a local registry.
+| ``{ENVPREFIX}_NOHTTPS``
+| This is relevant if you want to use a registry that doesn't support https. A
+  typical use-case for this variable is when using local registry, running on
+  the same machine as {Project} itself.
 
 Library
 -------
 
-**{ENVPREFIX}_LIBRARY** - Used to specify the library to pull from.
-Default is the currently selected :ref:`remote endpoint <endpoints>`.
+| ``{ENVPREFIX}_LIBRARY``
+| Used to specify the library to pull from.
+| Default is set to Sylabs' Cloud Library.
 
 Encryption
 ----------
 
-**{ENVPREFIX}_ENCRYPTION_PASSPHRASE** - Used to pass a plaintext
-passphrase to encrypt a container file system (with the ``--encrypt``
-flag). The default is empty.
+| ``{ENVPREFIX}_ENCRYPTION_PASSPHRASE``
+| Used to pass a plaintext passphrase to be used to encrypt a container file
+  system (in conjunction with the ``--encrypt`` flag). The default is empty.
 
-**{ENVPREFIX}_ENCRYPTION_PEM_PATH** - Used to specify the location of a
-public key to use for container encryption (with the ``--encrypt``
-flag). The default is empty.
+| ``{ENVPREFIX}_ENCRYPTION_PEM_PATH``
+| Used to specify the location of a public key to use for container encryption
+  (in conjunction with the ``--encrypt`` flag). The default is empty.
```

Comment:
I don't like how this new formatting looks without a hyphen.  I'm not sure exactly what the veritical bar formatting does, but based on how https://docs.sylabs.io/guides/3.11/user-guide/build_env.html#docker looks it will probably be OK if you just add a blank and a hyphen after each variable name.